python/spacetime_local/frame.py

```diff
@@ -161,30 +184,48 @@ def run(self):
         self.thread.start()
         self.thread.join()
 
+    def __clear(self):
+        self.__disconnected = False
+        self.__app.done = False
```

Comment:
Not sure what lines 188 and 189 are trying to do (with respect to clearing things)
The first tells frame that it is connected, and the second tells the app that its not done. Is that what you want to do?